pkg/utils/envutils/env.go

```diff
@@ -11,6 +11,11 @@ func IsEnvTruthy(envVarName string) bool {
 	return IsTruthyValue(os.Getenv(envVarName))
 }
 
+// EnvVal returns the value of an environment variable
+func GetEnv(envVarName string) string {
+	return os.Getenv(envVarName)
+}
+
```

Comment:
nit: I'm new and if this is a pattern we use at Solo, please ignore this comment. 

This has the same signature and doesn't change the result, it looks unnecessary. If I was working on something and saw an `GetEnv` call I would have to go look at this function to what it's doing only to find it isn't doing anything. Calling `os.Getenv` instead wouldn't have that overhead.

Reply:
I agree with this